app/controllers/api/v1/users_controller.rb

```diff
@@ -2,19 +2,24 @@
 
 module Api
   module V1
-    # Register users
-    class UsersController < ApplicationController
+    # Manages users
+    class UsersController < AuthenticatedController
+      def show
+        render json: current_user
+      end
+
       def create
         user = User.new(user_params)
 
         if user.save
-          # TODO: replace with real location
-          render json: user, status: :created, location: nil
+          render json: user, status: :created, location: api_v1_users_path
```

Comment:
`api` may be scope (not the namespace) in your `routes.rb` if you want to avoid `api_` prefix
It also applies to the `v1_`

Reply:
If I do so how do I versionate my API? How do I distinguish between v1 and v2?

Reply:
About `v1_` -- I think it is ok to keep `v1_users_path`

Reply:
I would like to see `users_path` without `api` nor `v1` to follow the rails way. But it is too much for this PR, let discuss it here